flow360/component/simulation/primitives.py

```diff
@@ -18,9 +18,9 @@ class ReferenceGeometry(Flow360BaseModel):
     - What about force axis?
     """
 
-    moment_center: Optional[LengthType.Point] = pd.Field()
-    moment_length: Optional[Union[LengthType.Positive, LengthType.Moment]] = pd.Field()
-    area: Optional[AreaType.Positive] = pd.Field()
+    moment_center: LengthType.Point = pd.Field()
+    moment_length: Union[LengthType.Positive, LengthType.Moment] = pd.Field()
+    area: AreaType.Positive = pd.Field()
```

Comment:
Then what is the moment center and moment length? You mean inherit from global setting?

Reply:
Yeah we should allow inherit from global setting, and there are default values right?

Reply:
[There was a issue](https://github.com/flexcompute/Flow360/pull/176#:~:text=In%20pydantic%20V1%20default_factory%20causes%20alot%20of%20issues%2C%20most%20importantly%20this%20one%20pydantic/pydantic%231717%20which%20breaks%20the%20updater%20which%20uses%20parse_obj%20extensively.%20There%20are%20other%20more%20subtle%20issues%20with%20default_factories%20on%20optional%20fields%20like%20confusing%20error%20messages.) I remember when setting default for this because of the unit system? But I added default and let's see.

Reply:
For example what unit to use.